tutorials/Tab_Navigation/README.md

```diff
@@ -0,0 +1,150 @@
+# Tabbing navigation and keyboard focus
+
+## What is covered
+
+In this tutorial, we will show you how to use tabbing navigation between components via keyboard shortcuts `tab` and `shift+tab`.
+
+## Simple example
+
```

Comment:
Could you please provide more explanations what's going on in this demo.

Reply:
Done.